src/formpack/schema/fields.py

```diff
@@ -47,14 +42,8 @@ def __init__(
         if source is not None:
             self.source = source
             self.analysis_question = True
-            self.analysis_type = kwargs.get('analysis_type')
-            self.analysis_path = kwargs.get('analysis_path')
             self.settings = kwargs.get('settings')
-            if self.analysis_type in [
-                ANALYSIS_TYPE_TRANSCRIPT,
-                ANALYSIS_TYPE_TRANSLATION,
-            ]:
-                self.language = kwargs['language']
```

Comment:
Using object-oriented programming and creating subclasses for analysis fields removes the need for these extra attributes